mypy/build.py

```diff
@@ -2421,6 +2415,25 @@ def find_module_and_diagnose(manager: BuildManager,
             raise CompileError(["mypy: can't find module '%s'" % id])
 
 
+def find_added_packages(suppressed: List[str],
+                        manager: BuildManager, options: Options) -> bool:
+    """Find if there are any newly added packages that were previously suppressed.
+
+    Exclude everything not in build for follow-imports=skip.
+    """
+    for dep in suppressed:
+        path = manager.find_module_cache.find_module(dep, manager.search_paths,
```

Comment:
This same call is performed at least twice. It would be less error-prone to move this in a function and update all identical calls to use the function instead.